starter-code/main.js

```diff
@@ -0,0 +1,129 @@
+const MongoDB = require('mongodb');
+const mongoClient = MongoDB.MongoClient;
+const clear = require('clear');
+const readline = require('readline');
+const rl = readline.createInterface({
+	input: process.stdin,
+	output: process.stdout
+});
+
+const url = `mongodb://localhost:27017/crunchbase`
+
+mongoClient.connect(url, (error, db) => {
+    if (error) {
+      console.log('Error trying to connect to the Database');
+      console.log(error);
+    } else {
+      console.log('Connection established correctly!! 😬');
+  
+      function mainMenu(){
+        clear();
+        printMenu();
+        rl.question('Type an option: ', (option) => {
+          switch(option){
+            case "1":
+              db.collection('companies').find({}, {name: 1, _id: 0}).toArray((error, result) => {
+                if (error) {
+                  console.log(error);
+                  rl.question(`\nType ERROR enter to continue: `, (answer) => { mainMenu() });
+                } else {
+                  console.log(result); 
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                }
+              });
+              break;
+            case "2":
+                let count = db.collection('companies').count();
```

Comment:
What about the callback? This code will pass to the next line before the query has finished. Furthermore, against what you think, you are not assigning the result of the query, careful with that.
```
db.collection('companies').count({}, (error, result) => { 
...
```